SECURITY.md

```diff
@@ -1,8 +1,32 @@
-# Security
+Thanks for helping make GitHub safe for everyone.
```

Comment:
Thanks fór All players AI gen

Reply:
[]()with:
  args: --user-agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.75 Safari/537.36" --no-progress .
  fail: true
  debug: false
  format: markdown
  jobSummary: true
  lycheeVersion: 0.14.2
  output: lychee/out.md
  token: ***
env:
  GITHUB_TOKEN: ***
Run # Cleanup artifacts from previous run in case it crashed
lychee
Run /home/runner/work/_actions/lycheeverse/lychee-action/c053181aa0c3d17606addfe97a9075a32723548a/entrypoint.sh
✗ [404] https://specifications.freedesktop.org/secret-service/ | Failed: Network error: Not Found
✗ [404] https://specifications.freedesktop.org/secret-service/ | Failed: Network error: Not Found
✗ [404] https://specifications.freedesktop.org/secret-service/ | Failed: Network error: Not Found
## Summary

| Status        | Count |
|---------------|-------|
| 🔍 Total      | 438   |
| ✅ Successful | 211   |
| ⏳ Timeouts   | 0     |
| 🔀 Redirected | 0     |
| 👻 Excluded   | 224   |
| ❓ Unknown    | 0     |
| 🚫 Errors     | 3     |

## Errors per input

### Errors in ./docs/configuration.md

* [404] [https://specifications.freedesktop.org/secret-service/](https://specifications.freedesktop.org/secret-service/) | Failed: Network error: Not Found

### Errors in ./docs/credstores.md

* [404] [https://specifications.freedesktop.org/secret-service/](https://specifications.freedesktop.org/secret-service/) | Failed: Network error: Not Found

### Errors in ./docs/environment.md

* [404] [https://specifications.freedesktop.org/secret-service/](https://specifications.freedesktop.org/secret-service/) | Failed: Network error: Not Found

Process completed with exit code 2.